src/api/web/api.js

```diff
@@ -138,10 +147,74 @@ module.exports = function(config, auth, storage) {
     res.redirect(base);
   });
 
+  route.get('/sidebar/(@:scope/)?:package', function(req, res, next) {
+    let packageName = req.params.package;
+    if (req.params.scope) {
+      packageName = `@${req.params.scope}/${packageName}`;
+    }
+
+    storage.get_package({
+      name: packageName,
+      keepUpLinkData: true,
+      req,
+      callback: function(err, info) {
+        res.set('Content-Type', 'application/json');
+
+        if (!err) {
+          info.latest = info.versions[info['dist-tags'].latest];
+          let propertyToDelete = ['readme', 'versions'];
+
+          _.forEach(propertyToDelete, ((property) => {
+            delete info[property];
+          }));
+
+          let defaultGravatar = 'https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mm';
+
+          if (typeof _.get(info, 'latest.author.email') === 'string') {
```

Comment:
`_.isString` does the job here

Reply:
Any reason to prefer use third part library than native operation even it using same code as this?

The source code of _.isString:
```js
function isString(value) {
      return typeof value == 'string' ||
        (!isArray(value) && isObjectLike(value) && baseGetTag(value) == stringTag);
}
```

Reply:
Mostly code consistency with the rest of the app. As you wrote at the end is just a wrap, so we are ok.